src/js/gi/components/PreviewBanner.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+class PreviewBanner extends React.Component {
+  render() {
+    if (!this.props.show) {
+      return null;
+    }
+
+    const headerDisplacementCSS = `
```

Comment:
It is possible to use these styles by having them in a class? instead of inline?